extensions/wikia/SpecialPromote/UploadVisualizationImageFromFile.class.php

```diff
@@ -57,7 +57,9 @@ public function checkWarnings(){
 	static public function UploadVerification($destName, $tempPath, &$error) {
 		$promoImage = PromoImage::fromPathname($destName);
 
-		if($promoImage->isValid() and ($promoImage->getCityId() == F::app()->wg->cityId)){
+		if ( ( $promoImage->isValid() && $promoImage->getCityId() == F::app()->wg->cityId ) ||
+			!PhalanxTitleBlock::checkTitle( Title::newFromText( $destName ), true )
+		) {
```

Comment:
![No](http://s2.quickmeme.com/img/27/27f6cb84a1b6bfbe17c47ce146d7e127ae307a1fbe327e5a847085782c0d0e1f.jpg) :smile: 

Please do add Phalanx logic in a completely different extension. SpecialPromote is on the list of features to sunset in SUS.